modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/src/main/java/com/liferay/portal/search/tuning/blueprints/admin/web/internal/display/context/EditBlueprintDisplayBuilder.java

```diff
@@ -223,13 +223,18 @@ private JSONObject _getSelectEntityJSONObject(String className) {
 
 				multiple = true;
 			}
+
+			/*
+			LPS-129056 - Address selector functionality
+
 			else if (className.equals(Organization.class.getName())) {
 				_prepareSelectEntityURL(
 					portletURL,
 					BlueprintsAdminMVCCommandNames.SELECT_ORGANIZATIONS);
 
 				multiple = true;
 			}
+			*/
```

Comment:
@oliv-yu Is this commented out because it throws an error?

Reply:
Ah not an error. Selecting multiple organizations had this issue when I was trying it out, if I remember correctly there was no submit button. 😓